vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -207,6 +208,7 @@ class HomeActivityViewModel @AssistedInject constructor(
                                     false
                             )
                     )
+                    updateIdentityServer(session)
```

Comment:
I don't think this is the right way to set up the id server
We should check whether  the id server is defined or not (without depending on cross signing)